website/source/docs/providers/chef/r/data_bag.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Creates and manages a data bag in Chef Server.
 ---
 
-# chef\_data\_bag
+# chef_data_bag
```

Comment:
It actually never did that. I don't know where that belief came from, but that's literally never happened 😄 . Here's an example of it working properly as-is: https://www.terraform.io/docs/providers/terraform/d/remote_state.html